src/main/webapp/player/simid_player.html

```diff
@@ -61,6 +61,9 @@
           ];
           adParams = JSON.stringify(adParamsObj);
           break;
+        case 'maps/simid_map.html':
```

Comment:
If you select simid_map, but then select something else again, doesn't this button stay visible?

Reply:
whoops, yep. Fixed that